source/renderer/app/containers/hardware-wallet/HardwareWallet.js

```diff
@@ -0,0 +1,100 @@
+// @flow
+import React, { Component } from 'react';
+import { observer, inject } from 'mobx-react';
+import MainLayout from '../MainLayout';
+import { buildRoute } from '../../utils/routing';
+import { ROUTES } from '../../routes-config';
+import type { InjectedContainerProps } from '../../types/injectedPropsType';
+import type { NavDropdownProps } from '../../components/navigation/Navigation';
+import HardwareWalletWithNavigation from '../../components/hardware-wallet/layouts/HardwareWalletWithNavigation';
+
+type Props = InjectedContainerProps;
+
+@inject('stores', 'actions')
+@observer
+export default class HardwareWallet extends Component<Props> {
+  static defaultProps = { actions: null, stores: null };
+
+  isActiveScreen = (page: string, item: NavDropdownProps) => {
+    const { app, wallets } = this.props.stores;
+    if (!wallets.active) return false;
```

Comment:
@DeeJayElly just a note! This will be changed because we will show active Hardware wallet here not regular active wallet. You can add store entry if you want. It is up to you.
Or at least define like here https://github.com/input-output-hk/daedalus/pull/2016/files#diff-3e8cbb77a45d70e7d8bd5f690537c3e1R52

Reply:
Ok got it.